trade-finance-manager-api/src/v1/validation/route-validators/validation-handler.js

```diff
@@ -0,0 +1,18 @@
+const { validationResult } = require('express-validator');
+
+/**
+ * Validation middleware, used on a per route basis to handle when validations are run on the inputs to an API route.
+ * @param {Object} req Request object
+ * @param {Object} res Response object
+ * @param {String} next Callback function name
+ */
+// eslint-disable-next-line consistent-return
```

Comment:
Can we please remove inline rule and resolve the linting error?

Reply:
agreed
`return next();` should to the trick